### PR TITLE
Small Update

### PR DESCRIPTION
**Pull request by sunhwapark**
Just a few basic changes for now.
I'll look through more of the story-line later this week.

### CODE REVIEW

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 1]
```diff
@@ -3208,35 +3208,35 @@ QUEST_LV_0300_20160811_003207	As you can tell by secret devices, what kind of a
...
+QUEST_LV_0300_20160811_003226	Dominance Magic is a powerful magic, it is enough to dispel demon barrier.{nl}You might face a slight problem but first, let's find the orb.
+QUEST_LV_0300_20160811_003227	The secret device in the supply room can only be opened by entering a password.{nl}The problem is the memory about the password, I don't have. {nl}
 QUEST_LV_0300_20160811_003228	Perhaps, there might be a part of my soul that remembers the password.. {nl}But at the moment, we have no time, we have to solve it ourselves.{nl}
 QUEST_LV_0300_20160811_003229	The password is comprised of numbers. {nl}There might be some clues around the device.
 QUEST_LV_0300_20160811_003230	I should think about how to use the Orb of Dominance Magic when we acquire it.{nl}It is very powerful. A powerful magic requires something of equal worth in return.
 QUEST_LV_0300_20160811_003231	You got the Orb of Dominance Magic.{nl}Nice job. {nl}
 QUEST_LV_0300_20160811_003232	Also, I figured out a way to use Orb of Dominance Magic.
 QUEST_LV_0300_20160811_003233	I did say something about ?쐅iving something in return?? right?{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: `?쐅iving` gibberish?

**Comment by sunhwapark** [comment]
There's 2 minor commits i've made to fix the symbols on to be added soon.
QUEST_LV_0300_20160811_003234
(the other line i can't recall)

**Review by Sunaries — CHANGES_REQUESTED** [review]

File: EnglishTranslation-master/QUEST_LV_0300.tsv (hunk 1 above)
Comment (sunhwapark) [reply EnglishTranslation-master/QUEST_LV_0300.tsv]: Yeah it's gibberish, it should say 'giving' , i have made a commit to fix it but i'm not able to push commits at the moment until i get home in a few hours. ( i lost net access when attempting to sync the aforementioned corrections )

**Comment by Sunaries** [comment]
Good to merge. 👍

**Pull request merged**